abstio/src/abst_data.rs

```diff
@@ -135,11 +155,7 @@ impl DataPacks {
     pub fn load_or_create() -> DataPacks {
         let path = crate::path_player("data.json");
         match crate::maybe_read_json::<DataPacks>(path.clone(), &mut abstutil::Timer::throwaway()) {
-            Ok(mut cfg) => {
```

Comment:
We used to do this before stuff in the UI would dynamically grab missing maps. No longer necessary. Release process calls `updater --minimal`, which also doesn't need this entry filled out.